dashboard/controller/classrooms/index.js

```diff
@@ -9,16 +9,16 @@ var superagent = require('superagent'),
 
 // init settings
 var ini = null;
-exports.init = function(settings) {
+exports.init = function (settings) {
```

Comment:
If changes in the code of this file does not affects the Assignment Feature, then reset the changes in this file.